README.md

```diff
@@ -134,6 +134,7 @@ For every single NPM package, we create a record in the Algolia index. The resul
     definitelyTyped: '@types/babel__core',
   },
   moduleTypes: ['unknown'], // esm | cjs | unknown
+  styleTypes: ['none'], // file extensions like css, less, scss or none if no style files present
```

Comment:
css | less | scss | none

Reply:
Technically any extension could be there as far as we do not filter files in `style` field

Reply:
Ok, fair.